src/plugins/aiudame.js

```diff
@@ -38,8 +39,11 @@ Provide a code example if needed. Answer should be in markdown format.
       .then((_) => {
         // console.log(_.data.choices[0].text.trim());
         emitter.emit(
-          "send:message",
-          _.data.choices[0].text.trim(),
+          eventTypes.OUT.sendMessage,
+          {
+            text: _.data.choices[0].text.trim(),
+            thread_ts: message.ts
```

Comment:
message.thread_ts por si ya estás en un thread
```suggestion
            thread_ts: message.thread_ts || message.ts
```